src/connectors/menu/connectMenu.js

```diff
@@ -137,74 +137,39 @@ export default function connectMenu(renderFn, unmountFn = noop) {
         return this.isShowingMore ? showMoreLimit : limit;
       },
 
-      refine(helper) {
-        return facetValue => {
+      init(initOptions) {
+        const { helper, createURL, instantSearchInstance } = initOptions;
+
+        this._createURL = facetValue =>
+          createURL(helper.state.toggleRefinement(attribute, facetValue));
```

Comment:
I think it's all right to leave this as either part of init or "if it doesn't yet exist" in getWidgetRenderState, not sure what we settled on in the other PR

Reply:
I also kept it in the `init` in my own PRs.